doc/nrf/ug_nrf9160.rst

```diff
@@ -259,7 +264,14 @@ To perform a FOTA upgrade, complete the following steps:
      You can hardcode the information in the application, or you can use functionality like AWS jobs to provide the URL dynamically.
 
 The full FOTA procedure depends on where the binary files are hosted for download.
-See the :ref:`aws_fota_sample` sample for a full implementation using AWS.
+
+You can refer to the following implementation samples:
+
+* :ref:`http_modem_delta_update_sample` - performs a full firmware OTA update of the modem.
+* :ref:`_http_full_modem_update_sample` - performs a delta OTA update of the modem firmware.
```

Comment:
This and the delta sample line has switched the descriptions

Reply:
Fixed